src/Plotly.NET.Interactive.Extension/Extension.fs

```diff
@@ -0,0 +1,29 @@
+﻿namespace Plotly.NET.Interactive.Extension
```

Comment:
I think using the namespace Plotly.NET.Interactive (dropping the '.Extension') is perfectly fine.